el-ognl example first commit

dragos-onac opened the pull request:
el-ognl example code moved as a subtree of the tiles-examples project

michaelsembwever commented:
LGTM

you'll need to make the commit message more detailed. 

by ""first commit" did you mean there was more commits intended for this PR?

dragos-onac commented:
Hi Michael, 
Thank you for the feedback!
If you like it, works and compiles, then there should be no more commits in this PR. 

Here is the full description (sorry for not adding it as a commit message):
- Added tiles-examples maven project as child of the tiles-parent
- Added tiles-elognl maven project as child of the tiles-examples project
- Added content of the tile-elognl project (container factory, tiles initialiser, tiles listener, associated unit tests and site.xml)

Should I do a new PR or try ammending this current one with the above description as described here https://help.github.com/articles/changing-a-commit-message/ ?
What do you recommend?

Thank you,
Dragos

michaelsembwever commented:
> Should I do a new PR or try ammending this current one with the above description as described here https://help.github.com/articles/changing-a-commit-message/ ?

Yes follow these instructions to improve your commit message. Thanks heaps :-)

Commit:
el-ognl example project initial (and full PR) commit contains:
- tiles-examples maven project as child of the tiles-parent
- tiles-elognl maven project as child of the tiles-examples project
- the tile-elognl project with container factory, tiles initialiser, tiles listener, associated unit tests and site.xml

dragos-onac commented:
Looks more decent now…
Should we close it?

## Thanks heaps ☺

Dragos

From: mck [mailto:notifications@github.com]
Sent: Thursday, July 21, 2016 12:49 PM
To: apache/tiles tiles@noreply.github.com
Cc: Dragos Onac dragos.onac@liss.co.uk; Author author@noreply.github.com
Subject: Re: [apache/tiles] el-ognl example first commit (#10)

Should I do a new PR or try ammending this current one with the above description as described here https://help.github.com/articles/changing-a-commit-message/ ?

Yes follow these instructions to improve your commit message. Thanks heaps :-)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHubhttps://github.com/apache/tiles/pull/10#issuecomment-234231337, or mute the threadhttps://github.com/notifications/unsubscribe-auth/ASiHf6JAWSvXc2m1385kLRmdtCCVb5SHks5qX1y9gaJpZM4JPj6P.

---- LISS Systems Ltd. 30th Floor, 40 Bank Street, Canary Wharf • London • E14 5NR Tel +44 20 31027905; Fax +44 20 31029511; Web http://www.liss.co.uk Liss Systems is a limited company registered in England and Wales. Registered number: 3260432. VAT Registered number: 681438714. Registered office: Devonshire House, 60 Goswell Road, London, EC1M 7AD. Please note that Liss Systems may monitor email traffic data. This message (and any associated files) is intended only for the use of the intended recipient(s) and may contain information that is confidential, subject to copyright or constitutes a trade secret. If you are not the intended recipient(s) you are hereby notified that any dissemination, copying or distribution of this message is strictly prohibited. If you have received this message in error, please notify us immediately by replying to the message and deleting it from your computer. Any views or opinions presented are solely those of the author and do not necessarily represent those of the company.

michaelsembwever commented:
Yes, I will merge and close. Thanks @dragos-onac
